src/rpn2_geoclaw_vec.f90

```diff
@@ -0,0 +1,403 @@
+! This code has been adapted for vectorization. 
+! For more details on how to use it, see the Makefile of the chile2010 example.
+
+!======================================================================
+       subroutine rpn2(ixy,maxm,meqn,mwaves,maux,mbc,mx, &
+                      ql,qr,auxl,auxr,fwave,s,amdq,apdq)
+!======================================================================
+!
+! Solves normal Riemann problems for the 2D SHALLOW WATER equations
+!     with topography:
+!     #        h_t + (hu)_x + (hv)_y = 0                           #
+!     #        (hu)_t + (hu^2 + 0.5gh^2)_x + (huv)_y = -ghb_x      #
+!     #        (hv)_t + (huv)_x + (hv^2 + 0.5gh^2)_y = -ghb_y      #
+
+! On input, ql contains the state vector at the left edge of each cell
+!     qr contains the state vector at the right edge of each cell
+!
+! This data is along a slice in the x-direction if ixy=1
+!     or the y-direction if ixy=2.
+
+!  Note that the i'th Riemann problem has left state qr(i-1,:)
+!     and right state ql(i,:)
+!  From the basic clawpack routines, this routine is called with
+!     ql = qr
+!
+!
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!                                                                           !
+!      # This Riemann solver is for the shallow water equations.            !
+!                                                                           !
+!       It allows the user to easily select a Riemann solver in             !
+!       riemannsolvers_geo.f. this routine initializes all the variables    !
+!       for the shallow water equations, accounting for wet dry boundary    !
+!       dry cells, wave speeds etc.                                         !
+!                                                                           !
+!           David George, Vancouver WA, Feb. 2009                           !
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+      use geoclaw_module, only: g => grav, drytol => dry_tolerance, rho
+      use geoclaw_module, only: earth_radius, deg2rad
+      use amr_module, only: mcapa
+
+      use storm_module, only: pressure_forcing, pressure_index
+      use statistics
```

Comment:
What is this module?

Reply:
It was a module I created to measure the execution time of each solver, but I removed it before this pull request. Apparently, I forgot to delete it from a couple of files, and my compilation tests did not complain because the .mod file was still present in my computer. I have already removed these "use statistics" lines from the code.